### PR TITLE
Docs updated

### PR DESCRIPTION
**Pull request by ernstk**

### CODE REVIEW

**Comment by ernstk** [comment]
I'm doing a downstream merge first before re-opening this pull request

File: windows/win_chocolatey.py [hunk 1]
```diff
@@ -117,4 +118,9 @@
   win_chocolatey:
     name: ARRv3
     source: webpi
+
+  #Install git using an alternate source:
+  win_chocolatey:
```
Comment (bcoca) [inline windows/win_chocolatey.py:123]: version added?

Comment (ernstk) [inline windows/win_chocolatey.py:123]: Merged in two weeks ago: https://github.com/ansible/ansible-modules-extras/pull/517

Comment (bcoca) [inline windows/win_chocolatey.py:123]: no, i meant it is missing the 'version_added' documentation

Comment (ernstk) [inline windows/win_chocolatey.py:123]: Ah I got you. Which is the next version? How can I tell when it will be released?

**Comment by gregdek** [comment]
@ernstk Thanks for submitting this pull request. Unfortunately, it appears that the underlying code has changed since your submission, and this PR now needs to be rebased. 

If you believe that this PR still applies, please rebase and we will put it into our community review process as soon as possible.

**Comment by gregdek** [comment]
@ernstk A friendly reminder: this pull request has been marked as needing your action, and it hasn't been updated in the last month. 

If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we'll keep it open pending your changes.

If we receive no indication from you within a week, we will close this pull request.

**Comment by gregdek** [comment]
Closed due to inactivity.

**Pull request closed without merging**